vignettes/detection_efficiency_vignette.Rmd

```diff
@@ -0,0 +1,542 @@
+#| fig-height: 5
+#| fig-width: 7
+ggplot() +
+  geom_point(
+    data = sample_detection_efficiency,
+    aes(x = distance_m, y = avg_percent_d),
+    size = 3
+  ) +
+  geom_hline(yintercept = c(0.10, 0.50, 0.90), linetype = 2) +
+  geom_smooth(
+    data = sample_detection_efficiency,
+    aes(x = distance_m, y = avg_percent_d),
+    method = "glm",
+    linewidth = 1,
+    method.args = list(family = binomial(link = "logit")),
+    colour = "#66c2a5", se = FALSE
+  ) +
+  geom_smooth(
+    data = sample_detection_efficiency,
+    aes(x = distance_m, y = avg_percent_d),
+    method = "glm",
+    linewidth = 1,
+    method.args = list(family = binomial(link = "probit")),
+    colour = "#fc8d62", se = FALSE
+  ) +
+  scale_y_continuous(breaks = seq(0, 1, 0.20)) +
+  theme_bw(base_size = 15) +
+  theme(
+    panel.grid = element_blank()
+  ) +
+  labs(
+    x = "Distance (m)",
+    y = "Detection efficency (%)"
+  )
+```
+
+Now that we have our distances, we will need to create locations at the given percentages (e.g., 50%) we want to deploy for the course of the study. Given the above data we will deploy range transmitters at 365-370 m away from the receiver. 
+
+# Redployment 
+
+## Create buffer ring 
+The first step in redeployment of the range transmitters is to calculate a buffer ring at the given distance estimated from the models (e.g., 370 m). We will use steps very similar to the deployment steps. 
+
+```{r, warning = FALSE}
+#| title: create redployment ring
+redeploy_loc <- st_buffer(dist = 370, rcv_osc_sf_12) %>%
+  st_cast("LINESTRING") %>%
+  mutate(
+    distance = 370
+  ) %>%
+  dplyr::select(distance, glatos_array, station_no, ins_serial_no, geometry)
+# now view redeployment rings
+mapview(rcv_osc_sf) +
+  mapview(redeploy_loc)
+```
+
+Now that we have the redeployment ring, we are going to pick 3 locations to potentially deploy range transmitters for the study period. We will then create an excel and gpx file of all the locations.
+
+First we transform our buffer rings into spatial points, view them using `mapview()` and filter out the points we want. Then save as an excel and/or gpx file.  
+
+```{r, warning = FALSE}
+#| title: tranform, view and filter
+
+redeploy_loc_pts <- redeploy_loc %>%
+  st_cast("POINT") %>%
+  mutate(
+    id = 1:nrow(.)
+  ) %>%
+  dplyr::select(id, distance:geometry)
+```
+View them in mapview, this part will might take a little bit to select each point/figure out which ones you want. 
+```{r}
+mapview(rcv_osc_sf) +
+  mapview(redeploy_loc_pts)
+```
+
+## Select redeployment location 
+Then filter out the points we want, this will change depending on your study site and what locations you want. We will transform the projection back to WGS 84 as this is likely what your gps or sonar will want. We will also add in a few columns to conform to GLATOS and OTN data standards and rearrange the column order. 
+```{r, results = 'hide'}
+redeploy_sites <- buffer_rings_pts %>%
```

Comment:
> Error: object 'buffer_rings_pts' not found

It's unclear if you intended to use 'buffer_rings_pts' from earlier script (Section 2) or if you meant to use 'redeploy_loc_pts' here; but this needs to be fixed.

Reply:
I have updated this to `redeploy_loc_pts` as that is what the correct object should be - sorry about that.